lib/sequent/core/event_store.rb

```diff
@@ -8,6 +8,11 @@ module Sequent
   module Core
     class EventStore
       include ActiveRecord::ConnectionAdapters::Quoting
+      if Gem.loaded_specs['activerecord'].version >= Gem::Version.create('7.2.0')
+        require 'active_record/connection_adapters/postgresql_adapter'
+        include ActiveRecord::ConnectionAdapters::PostgreSQL::Quoting::ClassMethods
```

Comment:
`#debrief` https://github.com/rails/rails/pull/51174/files#diff-9c10b17f0ee079f3e5294d17264e7e06986f6192cff165f6628fe99886c38c24L76

Looks like `ActiveRecord::ConnectionAdapters::Quoting#quote_table_name` that was used here, now depends on `#quote_column_name` that is set to raise `NotImplementedError` on the abstract implementation. Instead of the previous `to_s`.

So we need to have a way to point to the PostreSQL specific implementation instead.